OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -464,6 +466,10 @@ void trackBulkDownloadSwitchOn(@NonNull String courseId, int totalDownloadableVi
         String REMAINING_DOWNLOADABLE_VIDEOS = "remaining_downloadable_videos";
         // Subjects
         String SUBJECT_ID = "subject_id";
+        // A/A test
```

Comment:
Add docs about the keys and mention about Firebase remote config in it, better to give story id here

Reply:
done.